### ISSUE TITLE
No Chromium namespace sandboxing?

### ISSUE THREAD

**Issue by The-Compiler** [flathub/org.qutebrowser.qutebrowser#193]
When opening `chrome://sandbox` natively, I see:

![image](https://user-images.githubusercontent.com/625793/167441525-ce8e3ec8-3189-4c33-ad2f-3be3bf85b5e2.png)

while via Flatpak, I get:

![image](https://user-images.githubusercontent.com/625793/167441568-b2802ba4-7fd6-474e-8157-862a622fc3b2.png)

Do you have any idea why that's the case? Are namespaces just inherently unavailable with Flatpak? Or is this disabled somewhere (where)?

**Comment by tinywrkb** [comment]
No, we don't have these privileges in the sandbox. I believe that the responsibility of [bubblewrap](https://github.com/containers/bubblewrap) to drop them.

The Chromium Flatpak has a few [patches that add support for using Flatpak Spawn API to create sandboxed instances](https://github.com/flathub/org.chromium.Chromium/blob/master/patches/chromium/flatpak-Add-initial-sandbox-support.patch), so maybe we can apply them on QtWebEngine.

**Comment by tinywrkb** [comment]
https://github.com/flathub/io.qt.qtwebengine.BaseApp/pull/66

![swappy-20220510-215951](https://user-images.githubusercontent.com/44030034/167702657-aad13ef3-b680-48b2-99fd-bb8a4a595b51.png)

```
$ flatpak ps
Instance   PID     Application                 Runtime
894813645  3550274 org.qutebrowser.qutebrowser org.kde.Platform
775470672  3550050 org.qutebrowser.qutebrowser org.kde.Platform
```

I'm not sure what else to check.

**Comment by tinywrkb** [comment]
Fixed in the beta channel. Stable will be fixed with the 3.0.0 release and the switch to PyQt6.
